content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/clientlibs/site/js/tabs.js

```diff
@@ -99,17 +99,19 @@
 
             for (var i = 0; i < hooks.length; i++) {
                 var hook = hooks[i];
-                var capitalized = IS;
-                capitalized = capitalized.charAt(0).toUpperCase() + capitalized.slice(1);
-                var key = hook.dataset[NS + "Hook" + capitalized];
-                if (that._elements[key]) {
-                    if (!Array.isArray(that._elements[key])) {
-                        var tmp = that._elements[key];
-                        that._elements[key] = [tmp];
+                if (hook.closest("." + NS + "-" + IS) === that._elements.self) { // only process own tab elements
+                    var capitalized = IS;
+                    capitalized     = capitalized.charAt(0).toUpperCase() + capitalized.slice(1);
```

Comment:
@andreeadracea - this is a really minor one, but would you be able to remove the spaces introduced after this variable definitions to the `=` sign.